tests/integration/cli/conftest.py

```diff
@@ -0,0 +1,44 @@
+from distutils.dir_util import copy_tree
+from pathlib import Path
+
+import pytest  # type: ignore
+from click.testing import CliRunner
+
+import ape
+from ape import Project
+
+TEST_PROJECTS_FOLDER = Path(__file__).parent / "data" / "projects"
+
+
+@pytest.fixture(params=[p for p in TEST_PROJECTS_FOLDER.iterdir() if p.is_dir()])
+def project_folder(request, config):
+    project_source_dir = request.param
+    project_folder = config.PROJECT_FOLDER / project_source_dir.name
```

Comment:
Doesn't seem to cause problems but using `project_folder` as the function name and a variable name made me do a double take (to make sure it wasn't recursive).